packages/image_picker/image_picker_ios/ios/Classes/FLTImagePickerPhotoAssetUtil.m

```diff
@@ -20,6 +20,19 @@ + (PHAsset *)getAssetFromPHPickerResult:(PHPickerResult *)result API_AVAILABLE(i
   return fetchResult.firstObject;
 }
 
++ (NSURL *)saveVideoFromURL:(NSURL *)videoURL error:(NSError **)error {
+  if (![[NSFileManager defaultManager] isReadableFileAtPath:[videoURL path]]) {
+    return nil;
+  }
+  NSString *fileName = [videoURL lastPathComponent];
+  NSURL *destination = [NSURL fileURLWithPath:[self temporaryFilePath:fileName]];
+  [[NSFileManager defaultManager] copyItemAtURL:videoURL toURL:destination error:error];
+  if (error) {
```

Comment:
This will always be true (you want to be checking `*error`), so I'm confused how this code is passing tests.

Reply:
according to xcode, this statement is incorrect.